src/components/Progress/ProgressRing.js

```diff
@@ -75,7 +76,7 @@ const ProgressRing = (props) => {
       >
         <svg viewBox="-1 -1 2 2">{getPath(percComplete / 100, isForceComplete, variant)}</svg>
       </div>
-      <div className="slds-progress-ring__content">{iconEl}</div>
+      <div className="slds-progress-ring__content">{content}</div>
```

Comment:
The setup assistant uses text content within the progress nodes. As that can basically be anything, I added the possibility to add `children`.